networkx/algorithms/bipartite/covering.py

```diff
@@ -0,0 +1,67 @@
+"""
+********
+Covering
+********
+"""
+
+#    Copyright (C) 2015 by
+#    Nishant Nikhil <nishantiam@gmail.com>
+#    All rights reserved.
+#    BSD license.
+
+import networkx as nx
+from networkx.utils import not_implemented_for, arbitrary_element
+
+
+@not_implemented_for('directed', 'multigraph')
```

Comment:
Ah. I thought the both are the same.

@nishnik Apologies for asking you to make that change.